_posts/2016-08-28-mouse-dna-magnetic-fields.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: No permanent damage to mouse DNA from low magnetic fields
+author: RaoOfPhysics
+image:
+  feature: 2016-08-28-mouse-dna-magnetic-fields.jpg
+  credit: Nogas1974
+  caption: “DNA com GGN”
+---
+
+_Sustained exposure not found to cause permanent damage in cells in brain, kidney and liver that are involved in the transport, reabsorption or storage of iron._
+
+Since the 19th-century works of Hans Christian Ørsted, Michael Faraday, James Clerk Maxwell and others demonstrated that electricity and magnetism are in fact two sides of the same coin, we have harnessed this unified electromagnetism to provide for the energy needs of billions around the globe.
+However, it is thought that prolonged exposure to ambient electromagnetic (EM) fields can affect certain cells in the body, causing a variety of cancers.
+For this reason, several countries have adopted limits for EM exposure, with most European countries limiting flux densities to 0.1&nbsp;mT ([milliteslas](https://en.wikipedia.org/wiki/Tesla_(unit))).
+
+To test the effects of EM fields on biological cells in a controlled environment, researchers exposed mice to low-frequency fluxes of 0.1&nbsp;mT and 1.0&nbsp;mT over eight weeks, with control mice living in similar conditions but without the EM exposure.
+The frequency chosen for the research was 50&nbsp;Hz, which is the frequency of transmission for alternating current in most countries.
```

Comment:
Have added a bit about the power lines. Have also explained a bit about electromagnetism in the first line and changed "flux" to "fields" -- this was *mea culpa*. "Fields" refer to "flux densities".